src/pages/detail/tabs/intro/GroupIntroTab.tsx

```
@@ -6,28 +6,27 @@ import "prismjs/themes/prism-tomorrow.css";
-    group && group.notionPageId,
-    getNotionPage,
-  );
-
-  if (groupError) return <div>Group introduce page not found</div>;
-  if (recordMapError)
-    return <div>Notion page not found - Please Check your notion ID</div>;
 
-  if (!group || !recordMap) return <div>Loading...</div>; // TODO: Add loading UI
+interface GroupIntroTabProps {}
 
+const GroupIntroTab = ({}: GroupIntroTabProps) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)